forc-pkg/tests/invalid/patch_workspace_and_package/test_package/Forc.toml

```diff
@@ -0,0 +1,10 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "test_package"
+
+[dependencies]
+
+[patch.'https://github.com/fuellabs/sway']
+std = { git = "https://github.com/fuellabs/sway", branch = "test" }
```

Comment:
Will the tests using this fail if the branch `test` gets deleted?

Reply:
The test will just parse the toml files and request the list of patches. No further action will be performed.